app/src/main/kotlin/com/github/premnirmal/ticker/settings/WidgetSettingsFragment.kt

```diff
@@ -128,6 +130,21 @@ class WidgetSettingsFragment : BaseFragment(), OnClickListener {
               InAppMessage.showMessage(activity!!, R.string.layout_updated_message)
             })
       }
+
+      R.id.setting_widget_width -> {
+        showDialogPreference(R.array.widget_width_types,
+                DialogInterface.OnClickListener { dialog, which ->
+                  widgetData.setWidgetSizePref(which)
+                  setWidgetSizeSetting(widgetData)
+                  dialog.dismiss()
+                  broadcastUpdateWidget()
+                  /*if (which == 2) {
```

Comment:
is this needed? prefer to remove commented code if it is not used

Reply:
removed, missed while merging